### DISCUSSION TITLE
Re-Application gets rejected

### DISCUSSION THREAD

**Discussion by AeneasChristodoulou** [community/community#54513] [GitHub Education]
Hello guys, I had this exact same issue last year already, where we are supposed to enter mail-addresses supplied by our school for GitHub education. This year, I had to re-verify (did it with the same sort of document as last year) but it got rejected over me needing a school supplied mail address (which just don't exist).

Also, the ticket I had to open last year is gone, so I can't see what the solution was that they came up with last year.

I'm going to a Highschool in Hamburg, Germany, more details can follow if needed.

**Comment by samus-aran** [comment]
Hi @AeneasChristodoulou  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

It would also be recommended that either you or a faculty member provides evidence that students do not receive academic emails to help with the request. 👍

Hope this helps! Let us know how you get along.
